lib/html-composer.js

```diff
@@ -115,7 +115,8 @@ function renderTemplate() {
     }
 
     return Promise.resolve(multiplex.render(template, data))
-      .then(media.append(data));
+      .then(media.append(data))
+      .then(html => html); // media.append() returns Promise
```

Comment:
Ohhh dang it, this was a change from before I updated the scripts with a `Promise`. You're right, good catch. Can remove that line.